model/model.cc

```diff
@@ -18,6 +18,14 @@
 */
 #include "immsconf.h"
 
+#include <iostream>
```

Comment:
I like this ordering better, thanks.

Since you're already changing the includes, could you also sort them? I think that's easier to read and helps detect accidental duplicate includes.